TYPO3.Neos/Classes/TYPO3/Neos/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -167,6 +157,45 @@ protected function setPropertyLabels($nodeTypeLabelIdPrefix, array &$configurati
             if (isset($propertyConfiguration['ui']['aloha']) && $this->shouldFetchTranslation($propertyConfiguration['ui']['aloha'], 'placeholder')) {
                 $propertyConfiguration['ui']['aloha']['placeholder'] = $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, 'aloha.placeholder');
             }
+
+            if (isset($propertyConfiguration['ui']['help']['message']) && $this->shouldFetchTranslation($propertyConfiguration['ui']['help'], 'message')) {
+                $propertyConfiguration['ui']['help']['message'] = $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, 'ui.help.message');
+            }
+
+            $configurationThumbnail = isset($propertyConfiguration['ui']['help']['thumbnail']) ? $propertyConfiguration['ui']['help']['thumbnail'] : null;
+            $thumbnailUrl = $this->resolveHelpMessageThumbnail($nodeTypeName, $configurationThumbnail);
+            if ($thumbnailUrl !== '') {
+                $propertyConfiguration['ui']['help']['thumbnail'] = $thumbnailUrl;
+            }
+        }
+    }
+
+    /**
+     * Resolve help message thumbnail url
+     *
+     * @param string $nodeTypeName
+     * @param string $configurationThumbnail
+     * @return string $thumbnailUrl
+     */
+    protected function resolveHelpMessageThumbnail($nodeTypeName, $configurationThumbnail)
+    {
+        if ($nodeTypeName !== null) {
+            $thumbnailUrl = '';
+            if (isset($configurationThumbnail)) {
+                $thumbnailUrl = $configurationThumbnail;
+                if (strpos($thumbnailUrl, 'resource://') === 0) {
+                    $thumbnailUrl = $this->resourceManager->getPublicPackageResourceUriByPath($thumbnailUrl);
+                }
+            } else {
+                # look in well know location
```

Comment:
We could argue if this is a good convention or if using a convention in this case is good or not, but since it's a part of the Neos feature set, I would not like to remove it if we don't really have to. Even not at a major release. Because, every removed feature will create a hassle for users.